docs/proposals/corss-cluster-statefulset/README.md

```diff
@@ -0,0 +1,243 @@
+
+#### Story 1
+
+#### Story 2
+
+### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate? 
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+## Design Details
+
+
+### New API
+
+
+```golang
+type CrossClusterStatefulsetSet struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec is the desired state of the MultiClusterIngress.
+	// +optional
+	Spec CrossClusterStatefulsetSetSpec `json:"spec,omitempty"`
+}
+
+// CrossClusterStatefulsetSetSpec is the desired state of the CrossClusterStatefulsetSet.
+type CrossClusterStatefulsetSetSpec struct {
+	ResourceSelector policyv1alpha1.ResourceSelector `json:"resourceSelector,omitempty"`
+  // updateStrategy indicates the CrossClusterStatefulsetSetStrategy that will be
+	// employed to update CRs when a revision is made to Template.
+	UpdateStrategy CrossClusterStatefulSetUpdateStrategy
+}
+
+// StatefulSetUpdateStrategy indicates the strategy that the StatefulSet
+// controller will use to perform updates. It includes any additional parameters
+// necessary to perform the update for the indicated strategy.
+type CrossClusterStatefulSetUpdateStrategy struct {
+	// Type indicates the type of the CrossClusterStatefulSetUpdateStrategy.
+	Type CrossClusterStatefulSetUpdateStrategyType
+}
+
+// CrossClusterStatefulSetUpdateStrategyType is a string enumeration type that enumerates
+// all possible update strategies for the CrossClusterStatefulSet controller.
+type CrossClusterStatefulUpdateStrategyType string
+
+const (
+	// RollingUpdateCrossClusterStatefulSetStrategyType indicates that update will be
+	// applied to all CRs in the StatefulSet with respect to the StatefulSet
+	// ordering constraints. When a scale operation is performed with this
+	// strategy, new CR will be created from the specification version indicated
+	// by the CrossClusterStatefulSet's updateRevision.
+	RollingUpdateCrossClusterStatefulSetStrategyType CrossClusterStatefulSetUpdateStrategyType = "RollingUpdate"
+)
+
+```
+
+### Two part here for user CRD
+
+### propagate schedule result to all of member cluster for CRD
+
+propagate schedule result with CRD annotation,just like:
+
+```yaml
+# member1
+Name:         xline
+Namespace:    xline
+API Version:  apps.my.io/v1alpha1
+Kind:         XlineCluster
+Annotations:
+    corssclusterstatefulset.karmada.io/replicas: [
```

Comment:
What's the default behavior for the K8s default statefulset, should we set the start index for different clusters?